### PR TITLE
DCOS-18978: Update CNVS from 1.1.9 to 1.1.11

### PR DESCRIPTION
**Pull request by ashenden**
Update of CNVS to 1.1.11. Some variables were removed in 1.1.11, so necessary patches were applied so stylesheet can compile from LESS. Any styling issues from the bump to 1.1.11 will be adjusted in subsequent PRs against this branch (feature/ui-kit-updates).

Resolves: https://jira.mesosphere.com/browse/DCOS-18978

Previous PR that was closed when I renamed the branch: https://github.com/dcos/dcos-ui/pull/2433

### CODE REVIEW

**Comment by MatApple** [comment]
@ashenden is this supposed to be against `feature/DCOS-18977-ui-kit-consistency` or the feature branch you're working against?

**Comment by ashenden** [comment]
@MatApple this PR should be against `feature/DCOS-18977-ui-kit-consistency` which is the "working master".  The plan (as you and I discussed) is to continue to branch of `feature/DCOS-18977-ui-kit-consistency`, do whatever work, and then PR against `feature/DCOS-18977-ui-kit-consistency`.  We will continue to rebase `feature/DCOS-18977-ui-kit-consistency` onto `master` to ensure it is as up-to-date as possible.

**Comment by MatApple** [comment]
@ashenden got it. Thought there was another feature branch called `feature/ui-kit-updates` but looks like that's obsolete.

**Comment by ashenden** [comment]
@MatApple there was, I just renamed it to be more aligned with Epic I'm using to track work.

**Comment by MatApple** [comment]
alfred retest this please

**Comment by ahoskins** [comment]
These are legitimate test failures
screen shot 2017-10-10 at 11 46 24 am

**Comment by ashenden** [comment]
Thanks @ahoskins.  I was having a hard time figuring out where to see the specific tests failing, but was able to look into and resolve them after you posted the screenshot above.  The Jenkins UI can be hard to parse.

Looks like the tests are passing now.

**Comment by ashenden** [comment]
Woo! Thanks guys.
